addons/mail/models/ir_config_parameter.py

```diff
@@ -1,21 +1,35 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, models
+from odoo import api, models, _
+from odoo.exceptions import ValidationError
 
 
 class IrConfigParameter(models.Model):
     _inherit = 'ir.config_parameter'
 
+    def _check_mail_catchall_allowed_domains(self, value):
+        error_message = _('Invalid value for `mail.catchall.allowed_domains` (must be a comma seperated list of domains, e.g. example.com,example.org )')
```

Comment:
Maybe ``Value for `mail.catchall.allowed_domains` cannot be validated. It should be a comma separated list of domains e.g. example.com,example.org.`` (no parenthesis, and also there is a typo ^^ )